pages/dkp/kommander/2.3/install/air-gapped/catalog/index.md

```diff
@@ -166,22 +166,22 @@ See [Kommander Load Balancing][kommander-load-balancing] for more information.
 1.  Download the Kommander image bundle file:
 
     ```bash
-    wget "https://downloads.d2iq.com/dkp/v2.2.0/kommander-image-bundle-v2.2.0.tar.gz" -O kommander-image-bundle.tar.gz
+    wget "https://downloads.d2iq.com/dkp/v2.3.0/kommander-image-bundle-v2.3.0.tar" -O kommander-image-bundle.tar
```

Comment:
Is there some reason why we use -O here to rename the archive and remove the version number? (Note this happens in the next step too...)   I assume the `push image-bundle` command below that uses them doesn't care about the name...    

In other places on this page where we manipulate similar things with version numbers in the name we don't rename them so this just looks odd and inconsistent (and also leaves the user with some things on their disk that don't have versions anymore so if they want to keep them they'll probably rename them anyways....

Reply:
So, you are suggesting we have the command as: 
```suggestion
    wget "https://downloads.d2iq.com/dkp/v2.3.0/kommander-image-bundle-v2.3.0.tar" kommander-image-bundle.tar
```
What do you think @gracedo ?

Reply:
@cbuto maybe you can help us here?

Reply:
I think in some places it might make sense to rename the file, but I'd agree with @cneth and vote for consistency here. 

It would end up looking like 

```
wget "https://downloads.d2iq.com/dkp/v2.3.0/kommander-image-bundle-v2.3.0.tar"
```

then we would have to update all references to `kommander-image-bundle.tar` to be `kommander-image-bundle-v2.3.0.tar`